waspc/data/Generator/templates/server/src/types/index.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+import http from 'http'
+
+export type Application = express.Application
+
+export type Server = http.Server
```

Comment:
Thoughts on exporting type aliases like these? I would like people to be able to pass around/use the components of `ServerSetupFnContext` and still have proper typing, but _without_ having to import the actual `express`/`node` imports themselves. I think it makes it nice they can get it all from the same import path, but curious on the TS expert pros/cons here. The downside is they will still need to know it is actually an `express.Application` if they want to use it with other libraries, look up docs, etc. but I hope that is an ok use for aliases? Thanks!

Reply:
For example, see `function addCustomRoute(app: Application)` in `serverSetup.ts`.

Reply:
I like the reexport in our context, so users can import from `@wasp/types` 👍 

I would maybe write it a bit differently:
```
import { Application } from 'express';
import { Server } from 'http';

export type ServerSetupFn = (context: ServerSetupFnContext) => Promise<void>

 export type ServerSetupFnContext = {
   app: Application,
   server: Server,
 }
 
 export { Application } from 'express';
 export { Server } from 'http';
```

Reply:
Ok I will leave this to @sodic and @infomiho .
I need to take some time these days to learn TS but it is not yet that day :D!

Reply:
Sounds good, thanks @infomiho for a better convention to use!